pointcloud/point.go

```diff
@@ -75,8 +77,6 @@ type basicData struct {
 
 	hasValue bool
 	value    int
```

Comment:
Should we require value to be a uint8 to ensure serialization is always accurate?

Reply:
If anything, I think it's the serialization that should increase its number of bytes. I feel like `value` in basicPoint should be specified to be a `int32` rather than just an `int` (which can be 32 or 64 based on OS). That, or the serialization should specify in a header how many bytes the `value` will be.